buildSrc/src/main/groovy/io/micronaut/guides/IndexGenerator.groovy

```diff
@@ -275,7 +275,7 @@ class IndexGenerator {
     @CompileDynamic
     private static List<String> generateTags(GuideMetadata guide) {
         [
-            guide.tags +
+            guide.tags ?: [] +
```

Comment:
I think it's better to remove this so we make sure all the guides have at least one tag (besides the ones added automatically)

Reply:
I don't think we should force a guide to have a tag, I don't see any harm on making tags optional.